src/i2c.rs

```diff
@@ -190,21 +190,23 @@ impl<I2C, SCL, SDA> I2c<I2C, (SCL, SDA)> {
 
         // Configure for "fast mode" (400 KHz)
         // NOTE(write): writes all non-reserved bits.
-        i2c.timingr.write(|w| {
-            w.presc()
-                .bits(crate::unwrap!(u8::try_from(presc)))
-                .sdadel()
-                .bits(crate::unwrap!(u8::try_from(sdadel)))
-                .scldel()
-                .bits(crate::unwrap!(u8::try_from(scldel)))
-                .scll()
-                .bits(scl_low)
-                .sclh()
-                .bits(scl_high)
-        });
+        unsafe {
+            i2c.timingr().write(|w| {
+                w.presc()
+                    .bits(crate::unwrap!(u8::try_from(presc)))
+                    .sdadel()
+                    .bits(crate::unwrap!(u8::try_from(sdadel)))
+                    .scldel()
+                    .bits(crate::unwrap!(u8::try_from(scldel)))
+                    .scll()
+                    .bits(scl_low)
+                    .sclh()
+                    .bits(scl_high)
+            });
+        }
```

Comment:
```suggestion
        i2c.timingr.write(|w| {
            w.presc().set(crate::unwrap!(u8::try_from(presc)));
            w.sdadel().set(crate::unwrap!(u8::try_from(sdadel)));
            w.scldel().setcrate::unwrap!(u8::try_from(scldel)));
            w.scll().set(scl_low);
            w.sclh().set(scl_high)
        });
```

Reply:
Thanks

I have gone through all files and tried to replace a lot of the `.bits` with `.set` in c813ea1